source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -450,6 +450,10 @@ The above example configures the CDN route to have a default time-to-live of 15
 
 The CDN route service forwards cookie headers your app by default, so cookie-based authentication will work as expected. Other headers, such as HTTP auth, are stripped by default. If you need a different configuration, see the guidance on [forwarding headers](/deploying_services/use_a_custom_domain/#forwarding-headers).
 
+### Verifying protocols and cyphers
+
+You can find the TLS version and cyphers used on GOV.UK PaaS by using an SSL testing tool such as [SSL Labs](https://www.ssllabs.com/ssltest/). For your apps, check the hostname `cloudapps.digital` (or `london.cloudapps.digital` in the London region).
```

Comment:
This isn't quite right. Looking at the TLS ciphers on `(london.)cloudapps.digital` will show you those in use by our ALB. You need to test it against your custom domain with CloudFront to find the ones being used in communication with the client.

Reply:
Is that also the case for the TLS version? 🤔

Reply:
Yes